tests/Feature/ExampleTest.php

```diff
@@ -4,18 +4,262 @@
 
 use Tests\TestCase;
 use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\DB;
 
 class ExampleTest extends TestCase
 {
-    /**
-     * A basic test example.
-     *
-     * @return void
-     */
-    public function testBasicTest()
+    static $member_created_id;
+    static $project_created_id;
+    function testGetAllMembers()
     {
-        $response = $this->get('/');
+        $response = $this->json('GET', '/api/members');
+        $response->assertStatus(200);
```

Comment:
Please also confirm that the response body contains all the members that you are expecting. You can use the seeds to insert the data you need for testing:
https://laravel.com/docs/5.5/seeding

Reply:
i was try but if i keep running test may be the data update

Reply:
You can use a separate config file called `.env.testing` and specify a different database to use for testing. That way you don't have to worry about it.